### PR TITLE
Update README.md (Fix for #460)

### PR DESCRIPTION
**Pull request by Damian89**
Example is wrong (see #460)

### CODE REVIEW

**Comment by staoxiao** [comment]
Thanks~

**Comment by Damian89** [comment]
Hey @staoxiao 
I noticed while the current change seems to be formally correct, reranking doesnt work as expected - I guess some more adjustements need to be made, honestly I cant tell.

When using ` model_type='encoder', ` to merge a two reranking models and then getting scores for two sentences the output is:
` ` ` 
[
 [value1]
]
` ` `
This is the expected behaviour.

When using ` model_type='reranker' the output for one pair of query/sentence is:
` ` `
[
 [value1 value2]
]
` ` `

Can you tell me something acout this behaviour? Is this another bug? Can this be explained?
Best,
Damian

**Comment by staoxiao** [comment]
Hi, I run the command to merge two bge-reranker-base models, and the output is correct:
"""
from LM_Cocktail import mix_models, mix_models_with_data

# Mix fine-tuned model and base model; then save it to output_path: ./mixed_model_1
model = mix_models(
    model_names_or_paths=["BAAI/bge-reranker-base", "BAAI/bge-reranker-base"], 
    model_type='reranker', 
    weights=[0.5, 0.5],  # you can change the weights to get a better trade-off.
    output_path='./mixed_model_1')

import torch
from transformers import AutoModelForSequenceClassification, AutoTokenizer, BatchEncoding, PreTrainedTokenizerFast

tokenizer = AutoTokenizer.from_pretrained('./mixed_model_1')
model = AutoModelForSequenceClassification.from_pretrained('./mixed_model_1')
model.eval()

pairs = [['what is panda?', 'hi']]
with torch.no_grad():
    inputs = tokenizer(pairs, padding=True, truncation=True, return_tensors='pt', max_length=512)
    scores = model(**inputs, return_dict=True).logits.view(-1, ).float()
    print(scores)
"""
The scores is `tensor([-8.1544])`.
It's possible that there are some errors in your usage. Could you provide the code you used to test?
